crates/wasi-common/cap-std-sync/src/file.rs

```diff
@@ -121,6 +121,16 @@ impl WasiFile for File {
     async fn num_ready_bytes(&self) -> Result<u64, Error> {
         Ok(self.0.num_ready_bytes()?)
     }
+    fn isatty(&self) -> bool {
+        #[cfg(not(windows))]
+        {
+            rustix::io::isatty(&self.0)
+        }
+        #[cfg(windows)]
+        {
+            false
+        }
```

Comment:
Can this be cfg(unix) and cfg(not(unix)) instead?

Reply:
I changed the first to `cfg(unix)`, but I think `cfg(windows)` makes sense for the second. If we port to a non-Unix non-Windows platform in the future, we should figure out what `isatty` means for that platform.

Reply:
The isatty crate would figure that out, right?

Reply:
Not in its current form, because the atty crate currently has no interface for passing in a handle. It just supports literal stdin, stdout, and stderr. Possibly that will change in the future, and then we could simplify code like this.

Reply:
In this case yes, but in stdio.rs you are using atty on stdin only for windows. There the `#[cfg(windows)]` can be turned into `#[cfg(not(unix))]` without issues.

Reply:
Ah, ok. I've now made this change.